src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -117,7 +117,12 @@
             }
         }
 
-        .bcs-activity-feed-empty-state-illustration {
+        .bcs-empty-state-detail {
+            font-weight: bold;
+        }
+
+        .bcs-ActivityFeedEmptyStateFallbackIllustration,
```

Comment:
May want to wrap the illustration in a container (maybe `.bcs-empty-state-image`) instead.